packages/core/src/bag3d/core/assets/ahn/download.py

```diff
@@ -168,6 +168,21 @@ def laz_files_ahn3(context, config: LazFilesConfig, md5_ahn3, tile_index_ahn):
     laz_dir = ahn_laz_dir(context.resources.file_store.file_store.data_dir, 3)
     laz_dir.mkdir(exist_ok=True, parents=True)
     url_laz = tile_index_ahn[tile_id]["AHN3_LAZ"]
+    if url_laz is None or url_laz == "None":
+        logger.error(f"Tile {tile_id} does not have a download link for AHN3.")
+        return Output(
```

Comment:
We need to compute the file's hash (see below), because it needs to be added to the metadata table because the reconstruction depends on it.